pyproject.toml

```
@@ -0,0 +1,11 @@
+[tool.black]
+line-length = 100
+skip-string-normalization = 1
+
+[tool.isort]
+atomic = true
+line_length = 100
+multi_line_output = 3
+skip_gitignore = true
+known_first_party = ["validphys", "eko", "n3fit", "reportengine"]
+force_sort_within_sections = true
```

```suggestion
profile = "black"
force_sort_within_sections = true
```
See: https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

Not sure what exactly this does, or if it clashes with some of the other settings. It didn't complain though...